src/components/tvproviders/schedulesdirect.js

```diff
@@ -110,13 +106,12 @@ export default function (page, providerId, options) {
 
     function submitLoginForm() {
         loading.show();
-        sha256(page.querySelector('.txtPass').value).then(function (passwordHash) {
+        sha1(page.querySelector('.txtPass').value).then(function (passwordHash) {
```

Comment:
I still don't get why we do that in the client instead of on the server, but eh.

It feels like we should always send passwords in clear and expect HTTPS.